include/rogue/interfaces/stream/FrameIterator.h

```diff
@@ -254,6 +265,14 @@ namespace rogue {
                iter += csize;
                size -= csize;
             } while ( size > 0 && csize > 0 );
+#if 1
+			if ( size > 0 ) {
```

Comment:
I think we can leave this at:

if ( size > 0 ) throw(rogue::GeneralError("FrameIterator::toFrame","Frame to small to accept all data");